Cargo.toml

```diff
@@ -25,6 +25,7 @@ build = "build.rs"
 default = ["feat_common_core"]
 ## OS feature shortcodes
 macos = ["feat_os_macos"]
+openbsd = ["feat_os_openbsd"]
```

Comment:
sorry but i am not a fan of creating an exception for openbsd
I guess it is for utmpx?

Reply:
To disable support of `utmpx` (not supported on OpenBSD), I created a new specific feature "openbsd" not including `feat_require_unix_utmpx`, see #5596 for detailed explanations. @tertsdiepraam agreed my proposal.

Reply:
still, i don't think we want exceptions for every distro or unix :( 

what about creating a "fake" utmpx for openbsd ? and the command would return "unsupported"

Reply:
OK, I could try another solution:

1. close this PR and remove my work to add a specific feature "openbsd" => include openbsd in "unix" as others Unixes OS
2. create a fake utmpx support => unsupported for tools where it's required : `pinky`, `uptime`, `users` and `who`
3. in a second time, implement an "alternative utmpx" support for OpenBSD for `pinky`, `uptime`, `users` and `who` binaries.

Do you agree ?

Reply:
sounds good, thanks :)